### PR TITLE
DOCSP 39098  Recommend limiting DDL operations during a sync

### PR DESCRIPTION
**Pull request by ltran-mdb2**
## DESCRIPTION

Recommend limiting DDL operations during a sync

## STAGING

https://preview-mongodbltranmdb2.gatsbyjs.io/cluster-sync/DOCSP-39098/reference/mongosync/#data-definition-language--ddl--operations

## JIRA

https://jira.mongodb.org/browse/DOCSP-39098

## BUILD LOG

https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=66324f30b67dbf804aec9d92

## SELF-REVIEW CHECKLIST

- [ ] Does each file have 3-5 taxonomy facet tags?
  See the [taxonomy tagging instructions](https://wiki.corp.mongodb.com/display/DE/Taxonomy+tagging+instructions) and this [example PR](https://github.com/10gen/cloud-docs/pull/5042/files) 
- [ ] Is this free of any warnings or errors in the RST?
- [ ] Is this free of spelling errors?
- [ ] Is this free of grammatical errors?
- [ ] Is this free of staging / rendering issues?
- [ ] Are all the links working?

## EXTERNAL REVIEW REQUIREMENTS

[What's expected of an external reviewer?](https://wiki.corp.mongodb.com/display/DE/Reviewing+Guidelines+for+the+MongoDB+Server+Documentation)

### CODE REVIEW

File: source/reference/mongosync.txt [hunk 1]
```diff
@@ -334,6 +334,14 @@ To sync a source cluster to multiple destination clusters, use one
 information, see :ref:`Multiple Clusters Limitations
 <multiple-clusters-limitations>`. 
 
+Data Definition Language (DDL) Operations
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+DDL operations (operations that act on collections or databases) during
```
Comment (jmd-mongo) [inline source/reference/mongosync.txt:340]: [suggestion]

Consider starting this sentence with a verb. Perhaps: "Using DDL operations ...".

**Review by jmd-mongo — APPROVED** [review]
Thanks for this update, @ltran-mdb2! This one LGTM with an optional suggestion.

Thanks!
Joe

**Review by rohshar — CHANGES_REQUESTED** [review]
I think that we need to list out examples of DDL operations or link to what they are. They aren't explained anywhere in the mongosync documentation

**Comment by ltran-mdb2** [comment]
Thanks for your feedback @rohshar ! I added examples of DDL operations and linked to more information in the manual. What do you think?

**Comment by ltran-mdb2** [comment]
Done @rohshar !

**Review by rohshar — APPROVED** [review]

**Pull request merged**